### ISSUE TITLE
tutorial for Travis CI with GitHub

### ISSUE THREAD

**Issue by karenyyng** [drphilmarshall/GettingStarted#11]
Travis CI (Continuous Integration) is a software engineering tool for automating the running of unit tests and show a [build passing] tag on GitHub repos to show the test coverage status. (This also let us to know who to blame when things break). Is there any possibility that @drphilmarshall will consider adding one of his wonderful tutorials for this? I meant to learn about this for a while and will be glad to help out.

**Comment by drphilmarshall** [comment]
Yeah, that could be a nice addition! TBH I'd probably start a new, short
markdown page on this, and link to it from the Getting Started front page
(via the FAQ "Is there way I can continuously test my code on GitHub?"
